arithmetization/src/main/java/net/consensys/linea/zktracer/module/blockdata/Blockdata.java

```diff
@@ -55,30 +55,30 @@ public String moduleKey() {
 
   @Override
   public void traceStartBlock(final ProcessableBlockHeader processableBlockHeader) {
-    this.batchUnderConstruction = true;
-    this.wcp.additionalRows.push(this.wcp.additionalRows.pop() + TIMESTAMP_BYTESIZE);
+    batchUnderConstruction = true;
+    wcp.additionalRows.add(TIMESTAMP_BYTESIZE);
```

Comment:
Is `.add(x)` just an alias for `.push( bla.pop() + x )` ?

Reply:
no, because the underlying has changesd. Before it was a stack of int, now it's a `CountOnlyOperation`

Reply:
checks added

Reply:
I Finally removed the possibility to remove lineCounting. It's only usage was for blockData to remove 4 rows at the end of the block, and it coyuld lead to issue if the last tx of the block was adding less than 4 rows in WCP (which is is theory possible ...)